### DISCUSSION TITLE
Add further parameters to SSH config file

### DISCUSSION THREAD

**Discussion by gigmedia** [yioda/ansible-inventory-to-ssh-config#8] [Q&A]
Hi,

I would need to add further parameters to the ssh config file, such as `ProxyJump` and `ServerAliveInterval`.
How can I achieve that?
Thanks for any help!

Roberto

**Comment by yioda** [comment]
Hi @gigmedia 

I'm glad to hear your feedback!
These options you mentioned are good suggestions for improving this tool.
Let me think about how to implement it.
Or, would you mind providing a simple example?

Thanks again!
